packages/gestalt/src/PageHeader.js

```diff
@@ -31,48 +89,90 @@ type Props = {|
    * Page title. Will always be a level 1 heading. Content should be [localized](https://gestalt.pinterest.systems/pageheader#Localization).
    */
   title: string,
+  /**
+   * An optional thumbnail image to be displayed next to the title.
+   */
+  thumbnail?: Element<typeof Image>,
 |};
 
 /**
  * [PageHeader](https://gestalt.pinterest.systems/pageheader) is used to indicate the title of the current page, as well as optional actions.
  *
- * ![PageHeader light mode](https://raw.githubusercontent.com/pinterest/gestalt/master/cypress/integration/visual-test/__image_snapshots__/PageHeader-basic-sm%20%230.png)
- * ![PageHeader dark mode](https://raw.githubusercontent.com/pinterest/gestalt/master/cypress/integration/visual-test/__image_snapshots__/PageHeader-basic-dark-sm%20%230.png)
+ * ![PageHeader light mode](https://raw.githubusercontent.com/pinterest/gestalt/master/cypress/integration/visual-test/__image_snapshots__/PageHeader-items-secondaryAction-md%20%230.png)
+ * ![PageHeader dark mode](https://raw.githubusercontent.com/pinterest/gestalt/master/cypress/integration/visual-test/__image_snapshots__/PageHeader-items-secondaryAction-dark-md%20%230.png)
+ * ![PageHeader light mode](https://raw.githubusercontent.com/pinterest/gestalt/master/cypress/integration/visual-test/__image_snapshots__/PageHeader-thumbnail-badge-iconButton-sm%20%230.png)
+ * ![PageHeader dark mode](https://raw.githubusercontent.com/pinterest/gestalt/master/cypress/integration/visual-test/__image_snapshots__/PageHeader-thumbnail-badge-iconButton-dark-sm%20%230.png)
  *
  */
 export default function PageHeader({
+  badge,
+  dropdownAccessibilityLabel,
+  helperIconButton,
+  helperLink,
+  items,
   maxWidth = '100%',
   primaryAction,
   secondaryAction,
   subtext,
   title,
+  thumbnail,
+  borderStyle = 'none',
 }: Props): Node {
-  return (
-    <Box color="white" paddingX={8} width="100%">
-      <Flex flex="grow" justifyContent="center" maxWidth="100%">
-        <Flex alignItems="center" flex="grow" justifyContent="between" maxWidth={maxWidth}>
-          <Box marginEnd={4} minWidth={0} marginTop={2} marginBottom={2}>
-            <Heading size="500" lineClamp={1} accessibilityLevel={1}>
-              {title}
-            </Heading>
-            {subtext && (
-              <Box marginTop={2}>
-                <Text lineClamp={1}>{subtext}</Text>
-              </Box>
-            )}
-          </Box>
+  const { text: badgeText, tooltipText: badgeTooltipText } = badge || {};
 
-          {primaryAction && (
-            <Flex alignItems="center" gap={2} flex="none">
-              {secondaryAction}
-              {/* 48px height needed to maintain proper sizing when action is a Link */}
-              <Box alignItems="center" display="flex" marginTop={4} marginBottom={4} height="48px">
-                {primaryAction}
+  return (
```

Comment:
It looks like the breakpoints still need some work:
![Screen Shot 2022-04-04 at 10 50 36 AM](https://user-images.githubusercontent.com/12059539/161602767-5caac1d4-11e3-49be-8471-99dba1c69d1f.png)

Reply:
Breakpoints aren't based on container width but window width.

<img width="575" alt="Screen Shot 2022-04-04 at 4 01 58 PM" src="https://user-images.githubusercontent.com/10593890/161622604-7f58aaf0-1761-4f48-a0b8-36ac8ef046fa.png">
 be used